### ISSUE TITLE
CLI takes JSON input

### ISSUE THREAD

**Issue by lukaswolfrum** [amosproj/amos2022ss07-nft-playbook#36]
## User story
1. As a power user 
3. I want / need to be able to input the data via an JSON
4. So that I can mint multiple NFTs without having to run through the program over and over

## Acceptance criteria
* multiple NFTs can be minted at once
* The user can input NFT information from a JSON file
 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
